src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -976,22 +962,23 @@ def customizePerJob(self, job):
             jdl.append('+DESIRED_CMSDataLocations = "%s"\n' % ','.join(job['inputDatasetLocations']))
 
         # HighIO jobs
-        highIOjob = job.get('highIOjob')
-        if highIOjob:
+        if job.get('highIOjob'):
```

Comment:
Possible a useless suggestions, but since this code is polluted with `ifs`, how about something like:
`jdl.append('+Requestioslots = %d\n' % job.get('highIOjob', 0))`
?